python/ray/actor.py

```diff
@@ -259,7 +260,9 @@ def _bind(
         other_args_to_resolve = {
             PARENT_CLASS_NODE_KEY: actor,
             PREV_CLASS_METHOD_CALL_KEY: None,
+            BIND_INDEX_KEY: actor._ray_bind_index,
         }
+        actor._ray_bind_index += 1
```

Comment:
I think `_ray_dag_bind_index` is more descriptive here (`bind` is specific to Ray DAGs, which aren't a commonly used API).

Can you also add a comment explaining what it is?

Reply:
Done